rest_service/Entities/RecordingPlayer.cs

```diff
@@ -24,7 +24,7 @@ public override void Serialize(BsonSerializationContext context, BsonSerializati
         var document = new BsonDocument
             {
                 { "_id", value.Id },
-                { "Nickname", value.Nickname }
```

Comment:
Leave physical field to match the referenced collection/field. Keep as "Nickname"